.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "docs.particular.net",
+	"build": {
+		"dockerfile": "Dockerfile",
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/powershell:1": {}
+	},
+	"forwardPorts": [
+		55555
```

Comment:
5000 or 55555?

Reply:
That's the default port of the docstool if I remember correctly